utils/get_latest_copr_build

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env python
```

Comment:
Just curious - did you intentionally leave out .py extension from utils/get_latest_copr_build ?

Reply:
in this case yeah, because I don't really like to have tools with .py extension - I find it awkward to call anything that has still .py in its name

Reply:
we should not use `/usr/bin/env python` and we should use `/usr/bin/python3` nowadays, but anyway, as this is just helping script, I think it is ok.

Reply:
heh. Ok, just now I found that the library is just for Python3. So this should be changed as well (F29 still exists with Python2 as default ;-))